patternfly/patternfly-elements
# fix: issue with custom styles piercing shadow DOM rendering

tldr;

Content set had moved the tabs and accordion into the shadow DOM for controlled rendering but found that this strips all styles from the panels.  Not good!!  This patch reverts that change and keeps the tabs and accordion in the light DOM so that panel styles will continue to work.


### Preview

- [Content set preview](https://deploy-preview-1365--happy-galileo-ea79c4.netlify.app/elements/pfe-content-set/demo/) 

### Testing instructions

1. Regression testing on pfe-content-set
2. Add custom styles to the panel region and ensure those styles persist when the content set upgrades.


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist


### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
